examples/terraform/simple-install/groups.tf

```diff
@@ -4,6 +4,7 @@ resource "matchbox_group" "default" {
   profile = "${matchbox_profile.coreos-install.name}"
   # no selector means all machines can be matched
   metadata {
+    container_linux_oem = "${var.container_linux_oem}"
```

Comment:
simple-install is for the getting started tutorial targeted at first time users with bare-metal labs. I'd kinda prefer it be as minimal as possible and the module just have advanced options for those who need them